src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java

```diff
@@ -1237,7 +1231,7 @@ public DCTree parse(int pos) throws ParseException {
                 @Override
                 public DCTree parse(int pos) throws ParseException {
                     skipWhitespace();
-                    DCReference ref = reference(false);
+                    DCReference ref = reference();
```

Comment:
You seem to be opening up the possibility that `@throws` can be followed by a member, which is a regression.